templates/tsConstructs/enum.ejs

```diff
@@ -1,6 +1,6 @@
 <section class="code-doc ">
     <a name="<%-anchorId%>"></a>
     <h3 class="code-ref"><%-kindString%>: <%-name%></h3>
-    <% include comments %>
+    <%- include('comments'); %>
```

Comment:
Are you sure these include statements should be terminated by a semicolon? I would expect something like this:

```suggestion
    <%- include('comments') %>
```

 I am not very familiar with EJS, so it's very likely I am wrong.

Reply:
`include(...)` is just a JS function call. The trailing `;` is optional but I prefer to have it for consistency.